android/browserstack_sample.py

```diff
@@ -5,33 +5,8 @@
 from selenium.webdriver.support import expected_conditions as EC
 import time
 
-# Options are only available since client version 2.3.0
-# If you use an older client then switch to desired_capabilities
-# instead: https://github.com/appium/python-client/pull/720
-options = UiAutomator2Options().load_capabilities({
-    # Specify device and os_version for testing
-    "platformName" : "android",
-    "platformVersion" : "9.0",
-    "deviceName" : "Google Pixel 3",
-
-    # Set URL of the application under test
-    "app" : "bs://<app-id>",
-
-    # Set other BrowserStack capabilities
-    'bstack:options' : {
-        "projectName" : "First Python project",
-        "buildName" : "browserstack-build-1",
-        "sessionName" : "BStack first_test",
-
-        # Set your access credentials
-        "userName" : "YOUR_USERNAME",
-        "accessKey" : "YOUR_ACCESS_KEY"
-    }
-})
-
 # Initialize the remote Webdriver using BrowserStack remote URL
-# and options defined above
-driver = webdriver.Remote("http://hub.browserstack.com/wd/hub", options=options)
+driver = webdriver.Remote("http://hub.browserstack.com/wd/hub")
```

Comment:
```suggestion
driver = webdriver.Remote("http://localhost:4444/wd/hub")
```
I think we have something like this for demo purposes in automate.. can check with @alliv8 once
Also imo no need to remove options from here as they should be merged with caps from yml in case user has both..

Reply:
Hey @alliv8, which url should be used here? Also should I add options again?

Reply:
changed the url